DirectEmployers/MyJobs
# Pd 3537 assessment create custom filter components

## What's this PR do?
* custom FilterComponents (with optional multi-select functionality)
* FilterBar - which displays all selected filters and allows for removal of either individual or all selected filters.
* GET contacts from API

## Screenshots (if appropriate)
![screen shot 2017-08-01 at 4 59 05 pm](https://user-images.githubusercontent.com/29709626/28847653-0e135f82-76de-11e7-8913-0036f7ab3a14.png)

## How should this be manually tested?
1. Go to partner assessment
2. Try selecting various combinations of filters.
3. Note that the star filter is single-select
4. Try clearing filters individually, as well as by using the 'clear all' button

## Any background context you want to provide?
The styling is still preliminary and will require some polish...
